test/parallel/test-tls-fast-writing.js

```diff
@@ -24,11 +24,12 @@ const common = require('../common');
 if (!common.hasCrypto)
   common.skip('missing crypto');
 
+const fixtures = require('../common/fixtures');
 const assert = require('assert');
 const tls = require('tls');
 const fs = require('fs');
 
-const dir = common.fixturesDir;
+const dir = fixtures.fixturesDir;
 const options = { key: fs.readFileSync(`${dir}/test_key.pem`),
                   cert: fs.readFileSync(`${dir}/test_cert.pem`),
                   ca: [ fs.readFileSync(`${dir}/test_ca.pem`) ] };
```

Comment:
If you replace the three `readFileSyncs` (e.g. <code>fs.readFileSync(`${dir}/test_ca.pem`)</code> with `fixtures.readSync('test_ca.pem')`) you can remove the `const dir` line, and probably the `require('fs')` as well, although `make lint` will tell you that.

If you could push a commit to your branch making that change that'd be great!

Docs here: https://github.com/nodejs/node/tree/master/test/common#fixturesreadsyncargs-enc